src/devices/HCSR04/HCSR04.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.I2c.Drivers;
+using System.Device.Spi;
+using System.Device.Spi.Drivers;
+
+namespace Iot.Device.HCSR04
+{
+    public class Sonar : IDisposable
+    {
+        private readonly int _echo;
+        private readonly int _trigger;
+        private readonly GpioController _controller;
+
+        public Sonar(int triggerPin, int echoPin)
```

Comment:
Do you mind adding Doc comments to all of the public facing API? That way it is more obvious to consumers what does triggerPin and echoPin are for.

Reply:
Every HC-SR04 I've seen has "trig" and "echo" printed on the board so I think this is fine as is